src/workflow/password_enter.h

```diff
@@ -0,0 +1,29 @@
+// Copyright 2019 Shift Cryptosecurity AG
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef _PASSWORD_ENTER_H_
+#define _PASSWORD_ENTER_H_
+
+#include <ui/components/set_password.h> // for SET_PASSWORD_MAX_PASSWORD_LENGTH
```

Comment:
This include should be in `password_enter.c`.

Reply:
I put it here because the docstring of password_enter says the caller needs to use `SET_PASSWORD_MAX_PASSWORD_LENGTH`, so this makes it available to the caller (without having to look for the source and including a second header).

good/bad? :)

Reply:
I don't need it for my workflow in vim. But if it floats your boat ;).

The include should be in the c-file though. Files should include what they use.

Reply:
Since this is the current coding style, that it is acceptable to include files in the "matching" header file we will continue like this. The only drawback is that it is not possible to include the c-file (which could be necessary for unit testing).